From c268866d04e5c3973a42979eb1492c917c62e1c4 Mon Sep 17 00:00:00 2001
From: Honggang Li <honli@redhat.com>
Date: Thu, 18 Aug 2016 03:46:34 -0400
Subject: [PATCH] Add shebang for exp scripts

The shebang '#!/usr/bin/expect -f' will make opa-ff packages require
expect rpm package.

Signed-off-by: Honggang Li <honli@redhat.com>
---
 IbaTools/FastFabric/chassis_configure.exp          | 1 +
 IbaTools/FastFabric/chassis_fmconfig.exp           | 1 +
 IbaTools/FastFabric/chassis_fmcontrol.exp          | 1 +
 IbaTools/FastFabric/chassis_fmgetconfig.exp        | 1 +
 IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp | 1 +
 IbaTools/FastFabric/chassis_fmsecurityfiles.exp    | 1 +
 IbaTools/FastFabric/chassis_getconfig.exp          | 1 +
 IbaTools/FastFabric/chassis_reboot.exp             | 1 +
 IbaTools/FastFabric/chassis_upgrade.exp            | 1 +
 IbaTools/FastFabric/configipoib.exp                | 1 +
 IbaTools/FastFabric/ff_function.exp                | 1 +
 IbaTools/FastFabric/ipoibping.exp                  | 1 +
 IbaTools/FastFabric/load.exp                       | 1 +
 IbaTools/FastFabric/mpiperf.exp                    | 1 +
 IbaTools/FastFabric/mpiperfdeviation.exp           | 1 +
 IbaTools/FastFabric/reboot.exp                     | 1 +
 IbaTools/FastFabric/sacache.exp                    | 1 +
 IbaTools/FastFabric/switch_capture.exp             | 1 +
 IbaTools/FastFabric/switch_configure.exp           | 1 +
 IbaTools/FastFabric/switch_dump.exp                | 1 +
 IbaTools/FastFabric/switch_fwverify.exp            | 1 +
 IbaTools/FastFabric/switch_getconfig.exp           | 1 +
 IbaTools/FastFabric/switch_hwvpd.exp               | 1 +
 IbaTools/FastFabric/switch_info.exp                | 1 +
 IbaTools/FastFabric/switch_ping.exp                | 1 +
 IbaTools/FastFabric/switch_reboot.exp              | 1 +
 IbaTools/FastFabric/switch_upgrade.exp             | 1 +
 IbaTools/FastFabric/upgrade.exp                    | 1 +
 TestTools/basic.exp                                | 1 +
 TestTools/chassis.exp                              | 1 +
 TestTools/common_funcs.exp                         | 1 +
 TestTools/defs.exp                                 | 1 +
 TestTools/extmng.exp                               | 1 +
 TestTools/ib.exp                                   | 1 +
 TestTools/ibtools.exp                              | 1 +
 TestTools/install.exp                              | 1 +
 TestTools/mpi.exp                                  | 1 +
 TestTools/network.exp                              | 1 +
 TestTools/opa_to_xml.exp                           | 1 +
 TestTools/proc_mgr.exp                             | 1 +
 TestTools/sm_control.exp                           | 1 +
 TestTools/switch_methods.exp                       | 1 +
 TestTools/target.exp                               | 1 +
 TestTools/tools.exp                                | 1 +
 44 files changed, 44 insertions(+)

diff --git a/IbaTools/FastFabric/chassis_configure.exp b/IbaTools/FastFabric/chassis_configure.exp
index e3e69a7..7803946 100644
--- a/IbaTools/FastFabric/chassis_configure.exp
+++ b/IbaTools/FastFabric/chassis_configure.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmconfig.exp b/IbaTools/FastFabric/chassis_fmconfig.exp
index 4ee920f..c1b272c 100755
--- a/IbaTools/FastFabric/chassis_fmconfig.exp
+++ b/IbaTools/FastFabric/chassis_fmconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmcontrol.exp b/IbaTools/FastFabric/chassis_fmcontrol.exp
index 47f36de..335eec0 100755
--- a/IbaTools/FastFabric/chassis_fmcontrol.exp
+++ b/IbaTools/FastFabric/chassis_fmcontrol.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmgetconfig.exp b/IbaTools/FastFabric/chassis_fmgetconfig.exp
index 60a9064..2c87acd 100755
--- a/IbaTools/FastFabric/chassis_fmgetconfig.exp
+++ b/IbaTools/FastFabric/chassis_fmgetconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp b/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
index db61871..eae5ea8 100755
--- a/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
+++ b/IbaTools/FastFabric/chassis_fmgetsecurityfiles.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_fmsecurityfiles.exp b/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
index c8e74c6..e972108 100755
--- a/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
+++ b/IbaTools/FastFabric/chassis_fmsecurityfiles.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_getconfig.exp b/IbaTools/FastFabric/chassis_getconfig.exp
index a68c624..d3fd4f6 100644
--- a/IbaTools/FastFabric/chassis_getconfig.exp
+++ b/IbaTools/FastFabric/chassis_getconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_reboot.exp b/IbaTools/FastFabric/chassis_reboot.exp
index 234a6d4..fa57efd 100755
--- a/IbaTools/FastFabric/chassis_reboot.exp
+++ b/IbaTools/FastFabric/chassis_reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/chassis_upgrade.exp b/IbaTools/FastFabric/chassis_upgrade.exp
index 3b2dc1b..338c55c 100755
--- a/IbaTools/FastFabric/chassis_upgrade.exp
+++ b/IbaTools/FastFabric/chassis_upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/configipoib.exp b/IbaTools/FastFabric/configipoib.exp
index b064ffa..6422590 100755
--- a/IbaTools/FastFabric/configipoib.exp
+++ b/IbaTools/FastFabric/configipoib.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/ff_function.exp b/IbaTools/FastFabric/ff_function.exp
index 0014170..3322674 100755
--- a/IbaTools/FastFabric/ff_function.exp
+++ b/IbaTools/FastFabric/ff_function.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/ipoibping.exp b/IbaTools/FastFabric/ipoibping.exp
index d677d3c..6046382 100755
--- a/IbaTools/FastFabric/ipoibping.exp
+++ b/IbaTools/FastFabric/ipoibping.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/load.exp b/IbaTools/FastFabric/load.exp
index d7d6379..b362a06 100755
--- a/IbaTools/FastFabric/load.exp
+++ b/IbaTools/FastFabric/load.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/mpiperf.exp b/IbaTools/FastFabric/mpiperf.exp
index ce75a54..bda685c 100755
--- a/IbaTools/FastFabric/mpiperf.exp
+++ b/IbaTools/FastFabric/mpiperf.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/mpiperfdeviation.exp b/IbaTools/FastFabric/mpiperfdeviation.exp
index c124e5f..3ed1874 100755
--- a/IbaTools/FastFabric/mpiperfdeviation.exp
+++ b/IbaTools/FastFabric/mpiperfdeviation.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/reboot.exp b/IbaTools/FastFabric/reboot.exp
index c1c81a3..abe00f2 100755
--- a/IbaTools/FastFabric/reboot.exp
+++ b/IbaTools/FastFabric/reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/sacache.exp b/IbaTools/FastFabric/sacache.exp
index 6ad6946..00370b6 100755
--- a/IbaTools/FastFabric/sacache.exp
+++ b/IbaTools/FastFabric/sacache.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_capture.exp b/IbaTools/FastFabric/switch_capture.exp
index f24f23b..030eb60 100755
--- a/IbaTools/FastFabric/switch_capture.exp
+++ b/IbaTools/FastFabric/switch_capture.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_configure.exp b/IbaTools/FastFabric/switch_configure.exp
index 4e21ffa..e8ed7f8 100644
--- a/IbaTools/FastFabric/switch_configure.exp
+++ b/IbaTools/FastFabric/switch_configure.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_dump.exp b/IbaTools/FastFabric/switch_dump.exp
index 063bec2..f28d874 100755
--- a/IbaTools/FastFabric/switch_dump.exp
+++ b/IbaTools/FastFabric/switch_dump.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_fwverify.exp b/IbaTools/FastFabric/switch_fwverify.exp
index f1eae5e..130fccd 100755
--- a/IbaTools/FastFabric/switch_fwverify.exp
+++ b/IbaTools/FastFabric/switch_fwverify.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_getconfig.exp b/IbaTools/FastFabric/switch_getconfig.exp
index b313eb5..c71a195 100755
--- a/IbaTools/FastFabric/switch_getconfig.exp
+++ b/IbaTools/FastFabric/switch_getconfig.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_hwvpd.exp b/IbaTools/FastFabric/switch_hwvpd.exp
index 6570226..60cd72f 100755
--- a/IbaTools/FastFabric/switch_hwvpd.exp
+++ b/IbaTools/FastFabric/switch_hwvpd.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_info.exp b/IbaTools/FastFabric/switch_info.exp
index abfe292..165cae9 100755
--- a/IbaTools/FastFabric/switch_info.exp
+++ b/IbaTools/FastFabric/switch_info.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_ping.exp b/IbaTools/FastFabric/switch_ping.exp
index 2295eed..7388601 100755
--- a/IbaTools/FastFabric/switch_ping.exp
+++ b/IbaTools/FastFabric/switch_ping.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
diff --git a/IbaTools/FastFabric/switch_reboot.exp b/IbaTools/FastFabric/switch_reboot.exp
index 68cd0be..a763c97 100755
--- a/IbaTools/FastFabric/switch_reboot.exp
+++ b/IbaTools/FastFabric/switch_reboot.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/switch_upgrade.exp b/IbaTools/FastFabric/switch_upgrade.exp
index 6c010bb..da9acca 100755
--- a/IbaTools/FastFabric/switch_upgrade.exp
+++ b/IbaTools/FastFabric/switch_upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/IbaTools/FastFabric/upgrade.exp b/IbaTools/FastFabric/upgrade.exp
index 5aaf60e..fcfdfd4 100755
--- a/IbaTools/FastFabric/upgrade.exp
+++ b/IbaTools/FastFabric/upgrade.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/basic.exp b/TestTools/basic.exp
index 939cd69..6ac908d 100755
--- a/TestTools/basic.exp
+++ b/TestTools/basic.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/chassis.exp b/TestTools/chassis.exp
index 9ee7184..6bbc00f 100755
--- a/TestTools/chassis.exp
+++ b/TestTools/chassis.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/common_funcs.exp b/TestTools/common_funcs.exp
index 68d55d8..7f62eb8 100644
--- a/TestTools/common_funcs.exp
+++ b/TestTools/common_funcs.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/defs.exp b/TestTools/defs.exp
index c9e90e6..b13cdd1 100644
--- a/TestTools/defs.exp
+++ b/TestTools/defs.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/extmng.exp b/TestTools/extmng.exp
index e3ffc16..32b4ff0 100755
--- a/TestTools/extmng.exp
+++ b/TestTools/extmng.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/ib.exp b/TestTools/ib.exp
index 06c4a6e..8e949cd 100755
--- a/TestTools/ib.exp
+++ b/TestTools/ib.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/ibtools.exp b/TestTools/ibtools.exp
index 6d7e22e..95343e8 100644
--- a/TestTools/ibtools.exp
+++ b/TestTools/ibtools.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/install.exp b/TestTools/install.exp
index 9590013..9c47e4b 100755
--- a/TestTools/install.exp
+++ b/TestTools/install.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/mpi.exp b/TestTools/mpi.exp
index 120d54e..6112177 100755
--- a/TestTools/mpi.exp
+++ b/TestTools/mpi.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/network.exp b/TestTools/network.exp
index 93a9e2d..356ec16 100755
--- a/TestTools/network.exp
+++ b/TestTools/network.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/opa_to_xml.exp b/TestTools/opa_to_xml.exp
index 62d28a4..9c71826 100644
--- a/TestTools/opa_to_xml.exp
+++ b/TestTools/opa_to_xml.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/proc_mgr.exp b/TestTools/proc_mgr.exp
index 5dd1dee..3900a00 100755
--- a/TestTools/proc_mgr.exp
+++ b/TestTools/proc_mgr.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/sm_control.exp b/TestTools/sm_control.exp
index 7a778d7..cfbdeec 100755
--- a/TestTools/sm_control.exp
+++ b/TestTools/sm_control.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/switch_methods.exp b/TestTools/switch_methods.exp
index 9f13773..3b2b14f 100644
--- a/TestTools/switch_methods.exp
+++ b/TestTools/switch_methods.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/target.exp b/TestTools/target.exp
index 9460a67..9e8e7b3 100755
--- a/TestTools/target.exp
+++ b/TestTools/target.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
diff --git a/TestTools/tools.exp b/TestTools/tools.exp
index 850e789..aa7f825 100755
--- a/TestTools/tools.exp
+++ b/TestTools/tools.exp
@@ -1,3 +1,4 @@
+#!/usr/bin/expect -f
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015, Intel Corporation
-- 
2.7.4

